### PR TITLE
feat: getShortcuts (optionally) takes network and address

### PR DESCRIPTION
**Pull request by silasbw**
New wallet implementations can use the v2 endpoint. Old wallet implementations will continue to work. New shortcuts can require (network, address), but they won't show on old wallet implementations.

### CODE REVIEW

**Comment by silasbw** [comment]
example of new shortcut implementation here: https://github.com/valora-inc/hooks/pull/227/files#diff-4835a7ed6db6e3c11e4bc4acec639412877cce313fd9937d66a4b487550f3b82

File: src/api/index.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ function createApp() {
     }),
   )
 
-  const balancesRequestSchema = z.object({
+  const getHooksRequestSchema = z.object({
```
Comment (jeanregisser) [inline src/api/index.ts:30]: 👍

File: scripts/getShortcuts.e2e.ts [hunk 2]
```diff
@@ -2,7 +2,9 @@ import * as $ from 'shelljs'
 
 describe('getShortcuts', () => {
   it('should get shortcuts successfully', () => {
-    const result = $.exec('yarn getShortcuts')
+    const result = $.exec(
+      'yarn getShortcuts --address 0x2b8441ef13333ffa955c9ea5ab5b3692da95260d',
+    )
     expect(result.code).toBe(0)
   })
```
Comment (jeanregisser) [inline scripts/getShortcuts.e2e.ts:9]: Could we have 2 tests? one with and one without address?

**Review by jeanregisser — APPROVED** [review]
Great! 👍

**Pull request merged**